.pylintrc

```diff
@@ -0,0 +1,2 @@
+[Format]
+max-line-length=80
```

Comment:
I think that's a bit more friendly, yes. Ideally, I'd like to be able to enforce some rules (such as "add proper comments to your functions") and disable others (many of those related purely to style). Something like [this](https://github.com/ARMmbed/mbed-cli/blob/master/mbed/mbed.py#L17), although not that particular set of rules. With this kind of restriction in place, even the 8.5/10 rule becomes more reasonable IMHO. Although I admit that right know I don't know precisely what I want to enforce and what I don't want to enforce.